racetrack_client/racetrack_client/main.py

```diff
@@ -214,6 +220,15 @@ def _get_config():
     print(datamodel_to_yaml_str(client_config))
 
 
+@cli_get.command('auth-token')
+def _get_auth_token(
+    remote: str = typer.Option(default=None, show_default=False, help="Racetrack server's URL or alias name"),
+):
+    """Show all racetrack config values"""
```

Comment:
This docstring seems wrong. It should say "Show currently logged in auth token" or similar, no?